.travis.yml

```diff
@@ -22,22 +22,34 @@ before_install:
   - conda create -y -q -n test-env python=$TRAVIS_PYTHON_VERSION
   - source activate test-env
   - if [[ $TRAVIS_PYTHON_VERSION == 3* ]]; 
-        then 2to3 -nw {PACKAGE_NAME} > /dev/null; 
+        then 2to3 -nw giddy > /dev/null; 
     fi
 
 install:
   - conda install --yes pip
   - conda install --yes --file requirements.txt;
+  - cd ../ && git clone https://github.com/pysal/libpysal.git
+  - if [[ $TRAVIS_PYTHON_VERSION == 3* ]]; then 2to3 -nw libpysal/ > /dev/null; fi
+  - pip install -e ./libpysal
+  - git clone https://github.com/pysal/esda.git
+  - if [[ $TRAVIS_PYTHON_VERSION == 3* ]]; then 2to3 -nw esda/ > /dev/null; fi
+  - pip install -e ./esda
+  - git clone https://github.com/pysal/mapclassify.git
+  - if [[ $TRAVIS_PYTHON_VERSION == 3* ]]; then 2to3 -nw mapclassify/ > /dev/null; fi
+  - pip install -e ./mapclassify
+  - cd ./giddy
   - pip install -r requirements_dev.txt
 
 script:
   - python setup.py sdist >/dev/null
-  - nosetests --with-coverage --cover-package={PACKAGE_NAME};
+  - echo "check_stable=False" >giddy/config.py
+  - nosetests --with-coverage --cover-package=giddy;
 
 notifications:
     email:
         recipients:
-            {AUTHOR_EMAILS}
+                weikang9009@gmail.com
+                sjsrey@gmail.com
```

Comment:
Agree. Organizing them in a ticked list is shown in travis [notifications example](https://docs.travis-ci.com/user/notifications/).